### ISSUE TITLE
Unable to clear country value

### ISSUE THREAD

**Issue by julesrenaud** [ChrisEbert/strapi-plugin-country-select#8]
A recent change in the `Combobox` Strapi design system makes it so that the clear callback needs to be defined, else the clear icon doesn't show up and it's impossible to unset the value of the field.

I fixed the issue by adding this function inside the CountrySelect component:

```
(...)
const isValidValue = !value || parsedOptions.hasOwnProperty(value);

const handleClear = () => {
  onChange({
    target: {
      name: name,
      value: '',
      type: attribute.type,
    },
  });
};
(...)
```

...and by linking it to the Combobox component `clear` attribute:

```
(...)
<Combobox
    placeholder={placeholder && formatMessage(placeholder)}
    aria-label={formatMessage(intlLabel)}
    aria-disabled={disabled}
    disabled={disabled}
    value={isValidValue ? value : null}
    onChange={countryCode => onChange({ target: { name, value: countryCode, type: attribute.type }})}
    onClear={handleClear}
>
(...)
```

**Comment by SalahAdDin** [comment]
@ChrisEbert could you check this?
@julesrenaud Did you fix this?

**Comment by ChrisEbert** [comment]
Fixed with v1.0.3
